saos-persistence/src/main/java/pl/edu/icm/saos/persistence/model/JudgmentTextContent.java

```diff
@@ -88,7 +90,7 @@ public String getFilePath() {
     //------------------------ LOGIC --------------------------
     
     /**
-     * Tells if with content is associated with some external file.
+     * Tells if with content is associated some external file.
```

Comment:
The comment is confusing...
I suggest: Returns true if the judgment content is in external file.